typescript/common-resolvers/src/data/dataset-tutorial.ts

```diff
@@ -6,13 +6,15 @@ const origin =
     ? globalThis.location?.origin
     : "https://labelflow.ai";
 
-export const tutorialImageUrls = [
-  `${origin}/static/img/tutorial-image-1.jpg`,
-  `${origin}/static/img/tutorial-image-2.jpg`,
-  `${origin}/static/img/tutorial-image-3.jpg`,
-  `${origin}/static/img/tutorial-image-4.jpg`,
+export const tutorialImageDetails = [
```

Comment:
Can you remove all the code changes from this PR please ? We said I'd do this in my PR, and I did already in that commit https://github.com/Labelflow/labelflow/pull/252/commits/c8d6f54e4006d3cc8be8cd2a0f2516a9afa86c12

Reply:
It was specified here https://github.com/Labelflow/labelflow/issues/388 to do it ... 
Ok reverting the changes ...

Reply:
I removed the changes. This PR now only contains the images changes.